### DISCUSSION TITLE
Account Linking

### DISCUSSION THREAD

**Discussion by LilLegendGamerez** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#840] [General]
I just wanted to know if I could link my google account to this.

**Comment by sumeshir26** [comment]
Hello @LilLegendGamerez, why would you like this feature? What would be the use of doing this?

**Reply by LilLegendGamerez** [reply]
Maybe to connect to google home to be able to use the phrase such as, "Hey Google, turn on Living Room TV." or, "Turn on Valentino's Bedroom and set color to blue."

**Reply by im-coder-lg** [reply]
Wellz personal results aren't working for everyone here, it is a truly common issue. I think Google patched it up in their Assistant SDK.

**Reply by Melvin-Abraham** [reply]
@LilLegendGamerez As long as you are using the same account for both your Google Home and GAUDC's OAuth Token, you should be able to control your connected devices

**Comment by LilLegendGamerez** [comment]
Ok thank you
________________________________
